app/controllers/journeys/expenses/AdvertisingOrMarketingController.scala

```diff
@@ -45,11 +45,11 @@ class AdvertisingOrMarketingController @Inject() (override val messagesApi: Mess
     with I18nSupport {
 
   val businessId = "SJPR05893938418"
-  val taxYear = LocalDate.now.getYear
+  val taxYear    = LocalDate.now.getYear
```

Comment:
I assume this will be in different story passed in URL, why not set 2023 and provide // ticket number in comment when it will be replaced?